cypress/tests/core/basic/language-controlpanel.js

```diff
@@ -0,0 +1,27 @@
+describe('Language control-panel', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    cy.autologin();
+    cy.waitForResourceToLoad('@navigation');
+    cy.waitForResourceToLoad('@breadcrumbs');
+    cy.waitForResourceToLoad('@actions');
+    //cy.waitForResourceToLoad('@types'); // Uncommenting creates error: > Timeout resolving resource: @types (type undefined)
```

Comment:
@danalvrz That makes sense. The `@types` endpoint is used to get the schema for a content item when it is being viewed or edited, but since this is not a content view, it is not used. You can remove this line with the comment.